src/deepqmc/molecule.py

```diff
@@ -55,14 +55,12 @@ class Molecule:
     n_down: int
     n_shells: tuple
 
-    def __init__(self, *, coords, charges, charge, spin, unit='bohr', data=None):
+    def __init__(self, *, charges, charge, spin, coords=None, unit=None, data=None):
```

Comment:
what happens if the user passes the `coords` argument?

Reply:
The `coords` argument was kept here, to be able to keep the old `yaml` [config files](https://github.com/deepqmc/deepqmc/tree/87d844fde442689719a75356e4293a78889b0ca5/src/deepqmc/conf/hamil/mol) of the `Molecule`s. When using the CLI, and not specifying the sampler for each configuration explicitly, a single nuclear configuration is read from these config files. A more thorough approach would be to update the config files to separate the definition of the `Molecule` object from the definition of the nuclear coordinates. I'll come up with something along these lines soon.